t/helper/test-path-walk.c

```diff
@@ -49,8 +49,15 @@ static int emit_block(const char *path, struct oid_array *oids,
 		BUG("we do not understand this type");
 	}
 
-	for (size_t i = 0; i < oids->nr; i++)
-		printf("%s:%s:%s\n", typestr, path, oid_to_hex(&oids->oid[i]));
+	for (size_t i = 0; i < oids->nr; i++) {
+		struct object *o = lookup_unknown_object(the_repository,
+							 &oids->oid[i]);
+		printf("%s:%s:%s", typestr, path, oid_to_hex(&oids->oid[i]));
+
+		if (o->flags & UNINTERESTING)
+			printf(":UNINTERESTING");
+		printf("\n");
```

Comment:
I slightly more idiomatic way to achieve the same would be:

```c
		printf("%s:%s:%s%s\n", typestr, path, oid_to_hex(&oids->oid[i]),
		       o->flags & UNINTERESTING ? ":UNINTERESTING" : "");
```

But this is only a minor nit.